contracts/TokenListManager.sol

```diff
@@ -14,48 +14,43 @@ contract TokenListManager is ITokenListManager, AccessControl {
     using EnumerableSet for EnumerableSet.AddressSet;
 
     // Define roles for administrative control and user access.
-    bytes32 public constant ADMIN_ROLE = keccak256("ADMIN_ROLE");
-    bytes32 public constant USER_ROLE = keccak256("USER_ROLE");
+    bytes32 public constant TOKEN_MANAGER_ROLE = keccak256("TOKEN_MANAGER_ROLE");
 
     // Mappings for whitelisted and blacklisted tokens, categorized by token type.
     mapping(TokenType => EnumerableSet.AddressSet) private whitelistedTokens;
     mapping(TokenType => EnumerableSet.AddressSet) private blacklistedTokens;
 
-    // Contract constructor setting up initial admin role.
-    constructor() {
```

Comment:
Should grant `ADMIN_ROLE` and customized `TOKEN_MANAGER_ROLE` to `msg.sender` in constructor. Otherwise, no admin to grant roles.

Reply:
Solved